src/edu/psu/sweng500/team8/play/GameSession.java

```diff
@@ -106,12 +123,23 @@ public void doRedo() {
 	public void enterNumber(CellCoordinates cellCoordinates, int number) {
 		
 		Cell currentCell = board.getCell(cellCoordinates.getRowIndex(), cellCoordinates.getColumnIndex());
-		SudokuAction sudokuAction = new SudokuAction(cellCoordinates, currentCell.getNumber(), number);
 		
-		/* TODO Actual Board Update to be implemented */
+		/* TODO - come back after PencilMarkMaanger is implement
+		 * current assumption is that PecilMarkManager will be a snapshot of the current pencil mark status 

```

Comment:
Yea that's the plan I think. It would probably have a method like "updatePencilMarks" whenever a number is entered so that clears that number in the same row/column/block.